src/test/scala/chiselTests/experimental/SerializableModuleGeneratorSpec.scala

```diff
@@ -59,6 +59,15 @@ class SerializableModuleGeneratorSpec extends ChiselFlatSpec with Utils {
     circt.stage.ChiselStage.emitCHIRRTL(g.module())
   }
 
+  "SerializableModuleGenerator" should "be able to elaborate with D/I" in {
+    circt.stage.ChiselStage.emitCHIRRTL(
+      new RawModule {
+        val i0 = g.instance()
+        val i1 = g.instance()
+      }
+    )
```

Comment:
It would probably be good to include some sort of check here for what you expect to see in the FIRRTL

Reply:
OK! working on it.